server/server.go

```diff
@@ -37,10 +37,10 @@ func main() {
 		port = "3001"
 	}
 
+	log.Println("Listening at http://localhost:%s", port)
```

Comment:
oops 👼 I was wondering why you didn't use `Println`, I guess that explains it :)